gluster-exporter/conf/global.conf.sample

```diff
@@ -0,0 +1,8 @@
+{
+	"gluster-mgmt": "glusterd2",
+	"gluster-work-dir": "/var/lib/glusterd2/",
+	"gluster-binary-path": "/usr/sbin/glusterd2",
```

Comment:
Where is this used? did you mean `glustercli`? If yes, default it to "gluster"(without any path prefix)